packages/eslint-plugin-pf-codemods/src/rules/v6/pageRenameIsTertiaryNavGrouped/page-rename-isTertiaryNavGrouped.md

```diff
@@ -0,0 +1,17 @@
+### page-rename-isTertiaryNavGrouped [(#9948)](https://github.com/patternfly/patternfly-react/pull/9948)
+
+We've renamed the \`isTertiaryNavGrouped\` prop to \`isHorizontalSubnavGrouped\`.
```

Comment:
```suggestion
We've renamed the `isTertiaryNavGrouped` prop to `isHorizontalSubnavGrouped` on Page.
```

just to have the props as inline code in the README. We can leave the backticks escaped in the rule/tests, just add the "on Page." at the end of the message in those files as well.